students/grzegorz_bajorski/lesson_02_flow_control/adding_factorials.py

```diff
@@ -0,0 +1,10 @@
+n = int(input())
+suma = 0
+fractorial = 1
+
+for i in range(1,n+1):
+    fractorial *= i
+    suma += fractorial
+
+print(suma)
+
```

Comment:
Too many empty lines at the end of the file. According to PEP8 should be only one.